core/spec.md

```diff
@@ -2405,60 +2490,65 @@ The following describes the attributes of Registry model:
     Once set, any attempt to update the value MUST be silently ignored by
     the server.
 
-    When not specified, the default value is `false`.
-  - Type: Boolean.
-  - OPTIONAL.
+  - When not specified, the default value MUST be `false`.
 
 - `attributes."STRING".clientrequired`
-  - Indicates whether this attribute is a REQUIRED field for a client when
-    creating or updating an entity. When not specified the default value is
-    `false`. When the attribute name is `*` then `clientrequired` MUST NOT be
-    set to `true`.
-
-    During creation or update of an entity if this attribute is not
-    specified then an error MUST be generated.
-
   - Type: Boolean.
   - OPTIONAL.
+  - Indicates whether this attribute is a REQUIRED field for a client when
+    creating or updating an entity. During creation or update of an entity if
+    this attribute is not specified in the request then an error MUST be
+    generated.
+  - When not specified, the default value MUST be `false`.
+  - When the attribute name is `*` then `clientrequired` MUST NOT be set to
+    `true`.
 
 - `attributes."STRING".serverrequired`
+  - Type: Boolean
+  - OPTIONAL
   - Indicates whether this attribute is a REQUIRED field for a server when
-    serializing an entity. When not specified the default value is `false`.
-    When the attribute name is `*` then `serverrequired` MUST NOT be set to
+    serializing an entity.
+  - When not specified the default value MUST be `false`.
+  - When the attribute name is `*` then `serverrequired` MUST NOT be set to
```

Comment:
It feels like these two sentences should be split into separate bullets.

Reply:
done